docs/integrators/deep-history-squad.md

```diff
@@ -35,143 +39,339 @@ In the example above, the key `726573657276650000000a55544b2d326638306539` is de
+
+# Clone and build the node binary
+git clone https://github.com/multiversx/mx-chain-go.git --branch=$BINARY_TAG --single-branch --depth=1
+go build -C mx-chain-go/cmd/node -o $(pwd)/node
 ```
 
-Above, we've chosen (as an example) the archives in such a way as to reconstruct the history between the 15th and 25th of October (~10 days, ~120 _devnet_ epochs).
+Afterwards, open the configuration file `config/prefs.toml` and apply the configuration depicted above, in the section [**Observer configuration**](#observer-configuration).
+
+:::warning
+Editing the configuration is crucial for reconstructing and retaining the non-pruned history. Skipping this will result in a pruned database.
+:::
+
+Then, get (download) and extract **a recent daily archive (snapshot)** for the shard in question (e.g. shard 0). The daily archives are available to download **on request** ([Discord](https://discord.gg/multiversxbuilders) or [Telegram](https://t.me/MultiversXValidators)), from a cloud-based, _S3-compatible storage_ (Digital Ocean Spaces) - or you could fetch them from an existing regular full-history observer that you own. 
 
-Now, bootstrap the reconstruction as follows:
+Upon extracting the downloaded archive, you'll have a new folder in your workspace: `db`, which contains the blockchain data for the shard in question. This data should be moved to a new folder, named `import-db`. All in all, the steps are as follows:
 
 ```
-# Download the docker-compose configuration
-wget https://github.com/multiversx/mx-chain-deep-history/blob/main/docker-compose.yml
+# Ask for the full download link on Discord or Telegram:
+wget https://.../23-Jan-2024/Full-History-DB-Shard-0.tar.gz
+# This produces a new folder: "db"
+tar -xf Full-History-DB-Shard-0.tar.gz
+
+# "import-db" should contain the whole blockchain to be re-processed
+mkdir -p ./import-db && mv db ./import-db
+# "db" will contain the reconstructed database (empty at first)
+mkdir -p ./db
+```
+
+:::note
+Downloading the archives and extracting them might take a while.
+:::
+
+Now, the reconstruction workspace should look as follows (irrelevant files omitted):
 
-# Run the "bootstrap" Docker service
-DEEP_HISTORY_WORKSPACE=${HOME}/deep-history-workspace DOCKER_USER=$(id -u):$(id -g) docker compose \
-    --file ./docker-compose.yml \
-    --profile bootstrap \
-    --project-name deep-history-reconstruction up --detach
+```
+.
+├── config          # network configuration files
+├── import-db
+│   └── db          # blockchain data
+├── db              # empty
+└── node            # binary file, the Node itself
 ```
 
-If you prefer to wait in the current shell until the bootstrap finishes, omit the `--detach` flag.
+We are ready to start the reconstruction process :rocket:
 
-:::tip
-Downloading the archives and extracting them might take a while.
+```
+./node --import-db=./import-db --import-db-no-sig-check --log-level=*:INFO --use-log-view --log-save --destination-shard-as-observer 0
```

Comment:
`--use-log-view` is deprecated. Will be removed in the upcoming releases

Reply:
Fixed.